ext/intl/transliterator/transliterator.stub.php

```diff
@@ -34,7 +34,7 @@ public static function listIDs() {}
      * @return string|false
      * @alias transliterator_transliterate
      */
-    public function transliterate(string $subject, int $start = 0, int $end = -1) {}
+    public function transliterate(string $string, int $offset = 0, int $end = -1) {}
```

Comment:
I would keep `$start` here, for the symmetry with `$end`. This is an unusual API that uses start&end instead of offset&length.